languages/elixir/exercises/concept/agent/lib/community_garden.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Plot do
+  raise "Please implement the Plot struct"
+end
+
+defmodule CommunityGarden do
+  def start(opts) do
+    raise "Please implement the start/1 function"
+  end
+
+  def list_registrations(pid) do
+    raise "Please implement the list_registrations/1 function"
+  end
+
+  def register(pid, register_to) do
+    raise "Please implement the register/2 function"
+  end
+
+  def release(pid, plot_id) do
+    raise "Please implement the release/2 function"
+  end
+
+  def get_registration(pid, plot_id) do
+    raise "Please implement the get_registration/2 function"
+end
```

Comment:
There's a syntax error:
```suggestion
    raise "Please implement the get_registration/2 function"
  end
end
```